L1Trigger/Phase2L1GT/python/l1tGTAlgoBlockProducer_cff.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+import re
+
+algorithms = cms.VPSet()
+
+l1tGTAlgoBlockProducer = cms.EDProducer(
+    "L1GTAlgoBlockProducer",
+    algorithms = algorithms
+)
+
+def collectAlgorithmPaths(process) -> "tuple[cms.Path]":
+    str_paths = set()
+    for algorithm in algorithms:
+        algo_paths = re.sub(r'[()]'," " , algorithm.expression.value()).split()
+        for algo in algo_paths:
+            if algo in process.pathNames() :
+                str_paths.add(algo)
+    paths = set()
+
+    for str_path in str_paths:
+        paths.add(getattr(process, str_path))
+
+    return tuple(paths)
```

Comment:
Not necessarily a review comment, but I feel I should point out to other reviewers that as of right now the L1GT emulator menu is constructed in python as a series of modules and paths (changed from fixed XML menus), however, no agreed upon parser/cmsDriver implementation for this menu exists. This is one option the GT team has provided, but I think we are soliciting outside advice on what the most preferable way to handle it would be.

Reply:
To make out 12_5_2_patch1 based prototype work, I did this, mostly because I can get away with whatever in our cmsDriver code: https://github.com/cms-l1t-offline/cmssw/blob/315e0b6cbcc55bec5b5eecbe955f25f68db9daef/Configuration/Applications/python/ConfigBuilder.py#L1523-L1579